doc/usecases/current/membership_application.md

```diff
@@ -0,0 +1,42 @@
+# Use Case: Membership Application
+
+## Summary
+
+- **Scope:** Membership
+- **Level:** User Goal
+- **Actors:** User (Membership Applicant), Administrator (Chairman)
+- **Brief:** The user can apply for a membership (account)
+- **Assignee:** Thorben (Frontend), Lukas (Keycloak Installation)
+
+## Scenarios
+
+- **Precondition:**
+  The user has opened the app and is not logged in.
+- **Main success scenario:**
+  1. The user can click an register button which leads them to Keycloak to fill out:
+     - username
+     - first+last name
+     - email address
+  2. The user can look at public maps (read-only and without seeing addresses!)
+  3. The user can visit a 3-step form to apply for membership, with:
+     - user photo (for Nextcloud)
+     - telephone number (optional)
+     - website
+     - organisation
+     - experience (optional)
+     - photo of map
+     - location of map (not shown to public users)
+     - bank account
+     - which kind of membership:
+       - paid subscription membership
+       - unpaid membership the user can maintain via gathering a certain number of permacoins per year
+  4. An admin:
+     - removes the bank account data
```

Comment:
Why remove the data? In case of the "unpaid" membership, we don't even need the bank data at the start but later on in the case the threshold of permacoins was not reached. How do we get the information then?

Reply:
We always need the bank account data for the registration and it can be removed immediately afterwards from our database. The data will be stored by the bank (for Einzugsauftrag), and is not needed by the Verein afterwards.

Reply:
We are in the progress in finding out how exactly this could be done. We don't have a Vereinskonto yet. But anyway, this shouldn't affect the implementation.